rerun_py/rerun_sdk/rerun/log/experimental/blueprint.py

```diff
@@ -0,0 +1,160 @@
+# TODO(jleibs): docstrings
+from typing import List, Optional
+
+import rerun_bindings as bindings  # type: ignore[attr-defined]
+
+from rerun.recording_stream import RecordingStream
+
+
+def new_blueprint(
+    application_id: str,
+    blueprint_id: Optional[str] = None,
```

Comment:
with this many arguments, we should mandate that they are named!

```suggestion
def new_blueprint(
    application_id: str,
    *,
    blueprint_id: Optional[str] = None,
```